www/src/pages/components/alert/react/index.mdx

```diff
@@ -92,7 +92,11 @@ export const pageQuery = graphql`
         packageTable: thumbprintComponent(name: { eq: "@thumbtack/thumbprint-react" }) {
             ...PackageTableFragment
         }
+        # Get component props by path to component file.
```

Comment:
This seems like it should append the props to the bottom of the page, but it doesn't 🤔. I likely need some help on this to push it over the finish line.